examples/ping-pong/dummy.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"net"
+	"sync"
+	"time"
+)
+
+// Reference: https://github.com/pion/sctp/blob/master/association_test.go
+
+type dumbConn struct {
```

Comment:
Mind renaming this to something a little more self documenting? Maybe `disconnectedPacketConn` and add a doc comment for others in the future.

It would be nice if they don't have to jump in and read the `association_test.go` and they can understand what is going on.